src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import '../utils/veraibles';
```

Comment:
There seems to be a typo in the file path. It should be '../utils/variables' instead of '../utils/veraibles'.